src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
+        if (quantity < 0) {
+            throw new RuntimeException("Quantity can't be negative");
+        }
+        Storage.addFruit(fruit, quantity);
+    }
```

Comment:
The method `apply` should not be left with default access modifier. Follow the encapsulation principle and make sure to use an appropriate access modifier to restrict access as needed.